configure.ac

```diff
@@ -380,6 +380,10 @@ AS_IF([ test $? = 0 ],
        [AC_MSG_RESULT([yes])],
        [AC_MSG_ERROR([python setuptools is required])])
 
+AC_ARG_ENABLE([python-installation],
+  AS_HELP_STRING([--disable-python-installation], [Disable running setup.py install for swtpm_setup]))
```

Comment:
Is this now disabled by default, as stated in the commit message, or do I have to disable it explicitly?

Reply:
The conditional `PYTHON_INSTALLATION` is true unless `$enable_python_installation` is set to `no`, so one has to explicitly use `--disable-python-installation` (or `--enable-python-installation=no`), by default the variable will be unset and therefore he conditional is false.

Reply:
@diabonas Can you add your Signed-off-by to this commit, please.

Reply:
Done